bundles/AdminBundle/Resources/public/js/pimcore/settings/metadata/predefined.js

```diff
@@ -137,11 +137,11 @@ pimcore.settings.metadata.predefined = Class.create({
             {text: t("description"), sortable: true, dataIndex: 'description',
                 getEditor: function() { return new Ext.form.TextArea({}); },
                 renderer: function (value, metaData, record, rowIndex, colIndex, store) {
-                                if(empty(value)) {
-                                    return "";
-                                }
-                                return nl2br(value);
-                           }
+                    if (empty(value)) {
+                        return "";
+                    }
+                    return Ext.util.Format.htmlEncode(nl2br(value));
```

Comment:
The order of `htmlEncode()` and `nl2br()` is wrong here. 
The current code would produce: `&lt;br /&gt;` instead of `<br />`
So `nl2br(Ext.util.Format.htmlEncode(value));` would be the correct order 😊

Reply:
Done :)